sandbox-web/src/main/resources/application.yml

```diff
@@ -19,6 +19,23 @@ spring:
       naming:
         implicit-strategy: org.hibernate.boot.model.naming.ImplicitNamingStrategyLegacyJpaImpl
         physical-strategy: org.hibernate.boot.model.naming.PhysicalNamingStrategyStandardImpl
+  mail:
+    host: smtp.gmail.com
+    port: 587
+    username: exadelovichjhon
```

Comment:
It's a bad practice to commit credentials. Move usernmae/password to VM_OPTIONS

Reply:
done